### DISCUSSION TITLE
Ant bms 32s and tx/rx connector

### DISCUSSION THREAD

**Discussion by masterx1981** [syssi/esphome-ant-bms#77] [General]
Hi!
I have a fairly recent ant bms, and have the connector in photo:
![17009480793497282053814606869662](https://github.com/syssi/esphome-ant-bms/assets/76602320/b77f8f55-ec36-4b54-bd3d-cf8192adb582)
I have indentified the 4 wires that uses the stock display, and on the pcb are labeled V G A B.
V must be VCC, G GND, and the other 2 are rx and tx. The strange thing is that G not seem tied to ground (no low ohm reading on any place where on the pcb seem to be gnd), and on the other pins i can't get any voltage, in any condition (neither vcc).
Someone have some idea?
Thanks!

**Comment by masterx1981** [comment]
Maybe i've discovered something, the serial interface of the bms must be externally powered. I've a display for the bms, and on the v wire the display output 3v. The pack is a 32s, so i need to buy a buck from 100-120v to 5v. Then hope that an ams1117 (present on the esp01 adapter board) is enough to supply both esp01 and the serial interface of the bms.
I suppose that the serial ic on the bms is fully isolated, and need to be externally powered.
I was hoping to power the esp01 from the bms 3.3, but seem that no 3.3v is present on any external connector, and i not want to solder flying wires on the pcb. And also hope that 3.3v is as good as the 3v from the display, else i put some diodes in series that drop a bit the voltage.

**Reply by syssi** [reply]
You are on the right track! It's super important to apply 3.3V to the VCC pin of the connector.

**Reply by masterx1981** [reply]
I've already ordered some hlk-pm03 3.3v isolated smps, as soon as they arrive i play a little more.
Thanks!
